src/Microsoft.DotNet.CodeAnalysis.net45/Microsoft.DotNet.CodeAnalysis.net45.csproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
I'm not 100% sure, but you might need to add
```xml
<CopyNuGetImplementations>true</CopyNuGetImplementations>
```
to this property group to ensure that the packages restored for this project get deployed to the bin directory. @ericstj is that right?